app/main.py

```diff
@@ -1,8 +1,52 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
```

Comment:
In the requirements these attributes have a range of values they can accept. Let's check values and if they are incorrect, set some default value and is correct